content/_data/upgrades/2-235-3.adoc

```diff
@@ -0,0 +1,22 @@
+==== Repository Signing Key Update
+
+Beginning with Jenkins LTS release link:/changelog-stable/#v2.235.3[2.235.3], stable repositories will be signed with the same GPG keys that sign the weekly repositories.
+Administrators of Linux systems *must* install the new signing keys on their Linux servers *before* installing Jenkins 2.235.3.
+
+===== Debian/Ubuntu
+
+Update Debian compatible operating systems (Debian, Ubuntu, Linux Mint Debian Edition, etc.) with the command:
+
+---
+# wget -q -O - https://pkg.jenkins.io/debian-stable/jenkins.io.key | apt-key add -
+----
```

Comment:
```suggestion
---
```

Otherwise the GitHub Web browser does not work as expected

Reply:
Fixed in f6535c9a5b7131195af32bb7a842777d22e5b921 based on https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#source-code